Makefile

```diff
@@ -16,6 +16,8 @@ USE_GID ?= 10001
 DOCKER := $(shell which docker)
 DC=${DOCKER} compose
 
+SERVICES=db fakesentry redis-cache localstack statsd oidcprovider
```

Comment:
This list of services showed up in a couple of places in the file. In order to make sure we don't miss one in one of the places, it's better to centralize the list here.